server/service/osquery.go

```diff
@@ -2456,6 +2461,13 @@ func getQueryNameAndTeamIDFromResult(path string) (*uint, string, error) {
 		// 2017/legacy packs with the format "pack/<Pack name>/<Query name> are
 		// considered unknown format (they are not considered global or team
 		// scheduled queries).
+
+		// We can't infer the team from this and it can't be stored, but it's still valid
+		if strings.HasPrefix(path, "pack/") && strings.Count(path, "/") == 2 {
```

Comment:
The delimiter is always `/` for legacy packs?

Reply:
That is my understanding according to the comment above

Reply:
At least that's how fleet used to make them, if it's something else then we'll just get the same familiar `unknown format` error